openadapt/app/cards.py

```diff
@@ -55,7 +77,7 @@ def recording_prompt(options, record_button):
 
             dialog.open()
 
-    def terminate():
+    def terminate() -> None:
         global PROC
```

Comment:
Seems like `PROC` should be `process` since it's not a constant

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507